### PR TITLE
Adding Highlights to Changelog

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

**Comment by it33** [comment]
@lindsay, have your help reviewing?

**Comment by jwilander** [comment]
Assigning to CS as all release PRs must go through him.

**Comment by lfbrock** [comment]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
-The "UNDER DEVELOPMENT" section of the Mattermost changelog appears in the product's `master` branch to note key changes committed to master and are on their way to the next stable release. When a stable release is pushed the "UNDER DEVELOPMENT" heading is removed from the final changelog of the release. 
+Release date: 2015-12-16
 
-- **Release candidate anticipated:** 2015-12-10
-- **Final release anticipated:** 2015-12-16
+### Release Highlights
+
+#### iOS App
+
+- New [Mattermost iOS App](https://github.com/mattermost/ios) now available for iPhone, iPad, and iPod Touch
```
Comment (coreyhulen) [inline CHANGELOG.md]: @it33 the iOS app will not be in the app store by the 16th?  are we sill ok having it here.  It's not really coupled to the 1.3 release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+- New [Mattermost Push Notification Service](https://github.com/mattermost/push-proxy) encrypts notifications before sending them to your iOS devices
```
Comment (coreyhulen) [inline CHANGELOG.md]: @it33 Same thing.  The push notification service isn't part of the 1.3 bits.  Although 1.3 pushes messages to it.

Comment (coreyhulen) [inline CHANGELOG.md]: @it33 Also it doesn't encrypt them, it proxies them.

File: CHANGELOG.md (hunk 1 above)
Comment (it33) [inline CHANGELOG.md]: Yes, the notes don't mention app store at all, just the iOS repo

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
Going to manually rebase/merge

**Pull request closed without merging**